ansible/playbook/tasks/update.yml

```
@@ -104,6 +104,14 @@
 
     # Fix things that should be fixed before restarts.
 
+    - name: Stop systemd pmm-agent service, if running
+      systemd:
+        name: pmm-agent
+        state: stopped
+        enabled: no
+      when: not is_container
```

Let's rename this misnomer to something more semantic:
```suggestion
      when: not is_docker
```